data/presets/leisure/summer-camp.json

```
@@ -0,0 +1,39 @@
+{
+    "icon": "maki-campground",
+    "fields": [
+        "name",
+        "operator",
+        "address"
+    ],
+    "moreFields": [
+        "{@templates/contact}",
+        "gnis/feature_id-US",
+        "capacity",
+        "opening_hours",
+        "activities",
+        "dog"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "leisure": "summer_camp"
+    },
+    "terms": [
+        "camp",
+        "outdoor",
+        "recreation",
+        "scouts",
+        "youth",
+        "adventure"
```

Please sort the terms alphabetically:


```suggestion
        "adventure",
        "camp",
        "outdoor",
        "recreation",
        "scouts",
        "youth"
```